hyp3_testing/opera_compare.py

```diff
@@ -0,0 +1,272 @@
+"""Modified from the OPERA RTC comparison script created by JPL OPERA RTC team.
+Source: https://github.com/opera-adt/RTC/blob/main/app/rtc_compare.py
```

Comment:
Do we expect https://github.com/opera-adt/RTC/blob/main/app/rtc_compare.py to change, and is it worth devising some way to install it as a dependency here rather than copy-pasting?

Reply:
Sounds like it would be difficult to add their script as a dependency.